profile/src/v100/networks/network/network.rs

```diff
@@ -0,0 +1,348 @@
+use crate::prelude::*;
+
+/// Accounts, Personas, Authorized dapps for some Radix Network that user
+/// has created and interacted with.
+#[derive(
+    Serialize, Deserialize, Debug, Clone, PartialEq, Eq, Hash, uniffi::Record,
+)]
+pub struct Network {
```

Comment:
Rename `-> ProfileNetwork`?

Reply:
Renamed to `ProfileNetwork` in https://github.com/Sajjon/RadixWalletKit/pull/28/commits/247a59816e1c82ba21db8f5d95c1393e2fe055c7